scRNA-seq-advanced/02-celltype_assignment.Rmd

```diff
@@ -197,16 +286,32 @@ adt_df <- adt_df %>%
   )
 ```
 
-```{r}
+Now we will want to add the cell types we have assigned back to our original SCE object.
+We can do that by defining a new column name, `threshold_celltype` that will be added to the `colData` object.
```

Comment:
maybe `threshold_adt_celltype`? since it didn't use RNA (which we later say can also be used for thresholding!).